using_openshift/managing_nodes.adoc

```diff
@@ -0,0 +1,128 @@
+= Managing Nodes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+As an OpenShift administrator, you can manage link:../architecture/kubernetes_infrastructure.html#node[nodes] in your instance using the CLI.
+
+The CLI manages and provides information about known nodes in your instance by interacting with node objects. A node object is a representation of a node, which can be a physical or virtual system. The master uses this information to validate the node with health checks. See the https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/node.md#node-management[Kubernetes documentation] for more information on node management.
```

Comment:
[suggestion]
I don't think that the "CLI manages" anything (command line interface). You manage "somethiing" with the CLI. Is there a command that does that? So:

You can use <command> to manage and get information about known nodes in your instance by interacting with the node objects. A node object represents a node that can be a physical or virtual system. This information is used by the master to validate the node with health checks.